debug/fn_showDebugTextLocal.sqf

```diff
@@ -0,0 +1,21 @@
+#define PREFIX engimaTraffic
+#define COMPONENT debug
+#include "\x\cba\addons\main\script_macros_mission.hpp"
+
+/*
+ * Summary: Shows debug text on local client.
+ * Remarks:
+ *   if global variable "dre_var_CL_SilentDebugMode" is set to true, debug text will only be written to RTF-file and not shown on screen.
+ * Arguments:
+ *   _text: Debug text.
+ */
+
+if (!isNull player) then {
+   if (!ENGIMA_TRAFFIC_SilentDebugMode) then {
+      player sideChat (_this select 0);

```

Comment:
I personally find that `systemChat` is easier.

Reply:
yep 👍